### DISCUSSION TITLE
rose-pine, fzf-colors and selected entry is invisible

### DISCUSSION THREAD

**Discussion by mirsella** [ibhagwan/fzf-lua#1325] [Q&A] [answered]
hello

when using rose-pine light mode, the selected entry line is "invisible" because of the text color, no problem with the dark mode version

![1](https://github.com/ibhagwan/fzf-lua/assets/45905567/bc09334c-8a6c-463e-9398-d78b70a353e0)
![2](https://github.com/ibhagwan/fzf-lua/assets/45905567/3fd3a8b4-2a7a-4016-95e0-d26f235e08a4)

I tried looking into the highlight group, fzf-colors table and such, but it was too much to start as i have never touched theses configuration.

running :FzfLua highlights, there is a lot of entry where the text is "invisible" like when using fzflua. 
for example, just listing the first one not working:
@conceal
@diff.delta
@ibl.indent.char.1
@lsp.type.comment
@lsp.type.variable

Basically all the black color which seems to be default

thanks for any help

**Comment by ibhagwan** [comment, marked as answer]
This should be an issue to open in rose-pine, fzf-lua comes with a preset of highlights that work with the default colorscheme and many other color schemes, every single highlight group is configurable, to solve this specific issue you need to configure `fzf_colors`, the line “normal” color is defined by `fg+` , you can try setting `fzf_colors=true` to have fzf-lua automatically set the default highlight groups for you or link `FzfLuaFzfCursorLine` for the bg|fg of the current line.

**Reply by mirsella** [reply]
thanks for the explanation, have a great day !
